### PR TITLE
Catalyst Encryption Status Indicator

### PR DESCRIPTION
**Pull request by JaydenDev**

### CODE REVIEW

File: src/detection.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+function checkEncryption() {
+    var page = document.getElementsByClassName(“current”)[0].src
```
[the remark this answers is not in the document]
Comment (JaydenDev) [reply src/detection.js:2]: with?

[the remark this answers is not in the document]
Comment (MystPi) [reply src/detection.js:2]: Normal ones. `"`

[the remark this answers is not in the document]
Comment (JaydenDev) [reply src/detection.js]: so i changed that and still get
detection.js:2 Uncaught TypeError: Cannot read properties of undefined (reading 'src')

**Comment by JaydenDev** [comment]
gonna need help on this one
cant get it to work alone

**Comment by JaydenDev** [comment]
Since this is quite obviously a failure, closing PR.

**Pull request closed without merging**